src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Configuration/TargetFrameworkProjectConfigurationDimensionProviderTests.cs

```diff
@@ -71,7 +75,8 @@ public class TargetFrameworkProjectConfigurationDimensionProviderTests
             using (var projectFile = new MsBuildProjectFile(projectXmlTFMs))
             {
                 IProjectXmlAccessor _projectXmlAccessor = IProjectXmlAccessorFactory.Create(projectFile.Project);
-                var provider = new TargetFrameworkProjectConfigurationDimensionProvider(_projectXmlAccessor);
+                ITelemetryService _telemetryService = ITelemetryServiceFactory.Create();
```

Comment:
FYI, this happened in Roslyn and we lost all telemetry for 1 point release due to code change that cause telemetry MEF to be not wired in correctly. so we didn't fire any telemetry and there was no test that make sure telemetry is being actually fired.